lib/src/button/button.component.scss

```diff
@@ -123,14 +124,11 @@
         border-top-left-radius: 0;
 
         &::after {
-            display: inline-block;
-            width: 0;
-            height: 0;
-            vertical-align: middle;
-            content: "";
-            border-top: .3em solid;
-            border-right: .3em solid transparent;
-            border-left: .3em solid transparent;
+            display: flex;
+            height: 20px;
+            align-items: center;
+            font: normal normal normal 12px/1 FontAwesome;
+            content: "\f078";
```

Comment:
Will there ever be scenario where we'll need to use another icon/character for the split button toggle? If not, I'm guessing this could be revised to use the hc font when it's ready rather than FontAwesome.

Reply:
Totally agree, when #58 is ready, this should definitely get switched to the Catalyst font.  I noticed @corykon called out the same in one of his table.scss files.